backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/model/UserCreateModel.java

```diff
@@ -0,0 +1,19 @@
+package ca.gov.dtsstn.cdcp.api.web.v1.model;
+
+import java.util.List;
+import org.immutables.value.Value.Immutable;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.Email;
+
+@Immutable
+@Schema(name = "UserCreate")
+@JsonDeserialize(as = ImmutableUserCreateModel.class)
+public interface UserCreateModel {
+	
+	@Email
+	String getEmail();
+
+	List<UserAttributeModel> getUserAttributeModel();
```

Comment:
We should create a new, immutable, model for creating a user attribute, which consists of validation annotations to ensure that the name/value pair are not blank.

Reply:
Although `UserAttributeCreateModel` has been created, it is currently mutable (setter methods exist and fields not marked as `final`)